src/render.rs

```diff
@@ -16,29 +16,68 @@ pub(crate) fn render_group<F: FnMut(&mut Scene, &usvg::Node)>(
         let transform = transform * util::to_affine(&node.abs_transform());
         match node {
             usvg::Node::Group(g) => {
-                let mut pushed_clip = false;
-                if let Some(clip_path) = g.clip_path() {
-                    if let Some(usvg::Node::Path(clip_path)) = clip_path.root().children().first() {
-                        // support clip-path with a single path
+                let alpha = g.opacity().get();
+                let mix = match g.blend_mode() {
+                    usvg::BlendMode::Normal => match alpha < 1.0 {
+                        true => vello::peniko::Mix::Normal,
+                        false => vello::peniko::Mix::Clip,
+                    },
```

Comment:
Nit, but matching a Boolean feels strange.

```suggestion
                    usvg::BlendMode::Normal => if alpha < 1.0 {
                        vello::peniko::Mix::Normal
                    else { 
                        vello::peniko::Mix::Clip,
                    },
```